docs/index.md

```diff
@@ -42,16 +42,17 @@ gitStream is a workflow automation tool that enables you to use YAML configurati
 
 	That's it! Now sit back and watch gitStream run automation rules on your next PR.
 
-	!!! tip "Tip: Install gitStream for your entire organizaton"
+	!!! tip "Tip: Install gitStream for your entire organization"
 
 		gitStream can be installed for one repo, specific repos, or all repos in your organization. We recommend installing for **all** because it will ensure all new repos are able to use gitStream. You can change this setting at any time in the future.
 
 
 === ":fontawesome-brands-gitlab: GitLab"
 
-	Please contact LinearB 
-
-	:material-email-outline: [customers@linearb.io](mailto:customers@linearb.io)
+	1. Create an account in the <a href="https://app.linearb.io/sign-up" target="_blank">LinarB app</a>.
```

Comment:
1. There is a typo, should change to `LinearB`
2. I think this text will be better "Create a free LinearB account"

Reply:
1. fixed
2. I thought about it, but this is not only for free accounts.

Reply:
but the account is free, its better to say free. You can say that if you already have an account you can skip that.

Reply:
How about: "Log in, or create a free account in the LinearB app"?

Reply:
prefect ( in → on, i.e. "on the LinearB app.")